zookeeper-metrics-providers/zookeeper-prometheus-metrics/pom.xml

```diff
@@ -33,6 +33,7 @@
 
   <properties>
       <prometheus.version>0.9.0</prometheus.version>
+      <datasketches.version>4.2.0</datasketches.version>
```

Comment:
Do you know if the lock contention issue is fixed in the latest prometheus client library?

Reply:
@li4wang 
I conducted a benchmark for `Summary#observe` performance between versions `0.16.0` and `1.3.5` (latest) of the Prometheus Java client library. The result shows minimal improvement:
```
Benchmark                        (detailLevel)  (version)   Mode  Cnt  Score   Error   Units
SummaryBenchmark.summaryObserve          BASIC        1.x  thrpt    3  1.027 ± 0.446  ops/us
SummaryBenchmark.summaryObserve          BASIC        0.x  thrpt    3  1.041 ± 0.843  ops/us
SummaryBenchmark.summaryObserve       ADVANCED        1.x  thrpt    3  1.009 ± 0.671  ops/us
SummaryBenchmark.summaryObserve       ADVANCED        0.x  thrpt    3  0.967 ± 0.979  ops/us
```

Full result: https://gist.github.com/Shawyeok/4a7ef034baa187b27a9321145d2f05c9